test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -297,6 +297,55 @@ test.describe( 'Cover', () => {
 
 		expect( isClickable ).toBe( false );
 	} );
+
+	test( 'can use focal point picker to set the focal point of the cover image', async ( {
+		editor,
+		coverBlockUtils,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+		const coverBlock = editor.canvas.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+
+		await coverBlockUtils.upload(
+			coverBlock.getByTestId( 'form-file-upload-input' )
+		);
+
+		await editor.selectBlocks( coverBlock );
+
+		const focalPointLeft = page.getByRole( 'spinbutton', {
+			name: 'Focal point left position',
+		} );
+
+		const focalPointTop = page.getByRole( 'spinbutton', {
+			name: 'Focal point top position',
+		} );
+
+		await focalPointLeft.fill( '20' );
+		await focalPointTop.fill( '20' );
```

Comment:
```suggestion
		await focalPointLeft.fill( '20' );
		await focalPointTop.fill( '30' );
```

Let's use different values ​​to check each one is applied correctly.